### ISSUE TITLE
Max retries exceeded. Unstructured API is stuck.

### ISSUE THREAD

**Issue by Neel-132** [Unstructured-IO/unstructured#3169]
Unstructured API is stuck and it just logs "Starting  new HTTPS Connection..."

**Environment Info**
PyMuPDF==1.24.4
unstructured[all-docs]==0.14.0
nltk==3.8.1
pandas==2.2.2
llama-index==0.10.38
llama-parse==0.4.3
PyPDF2==3.0.1
fastapi==0.111.0
minio==7.2.7
uvicorn==0.29.0
psutil==5.9.8
boto3==1.34.109
botocore==1.34.109

The same error occurred some time back as well but I fixed it by making split_pdf_page = True. But right now its again stuck

**Comment by awalker4** [comment]
Hi there - is this for the paid api, free api, or a self host? We're experiencing some network issues on the paid api right now, we're working to resolve ASAP/

**Comment by awalker4** [comment]
Yep! Take a look at the retries documentation [here](https://docs.unstructured.io/api-reference/api-services/sdk#retries). In your screenshot, the error is `no healthy upstream` which was due to our network issues just now. You can adjust the retry config to change how long the client tries before giving up.
